randovania/games/prime1/json_data/Phazon Mines.txt

```diff
@@ -855,7 +915,7 @@ Extra - asset_id: 4272124642
           Any of the following:
               Morph Ball Bomb
               All of the following:
-                  # https://www.youtube.com/watch?v=rDCTXaNmypI
+                  # Original video: https://www.youtube.com/watch?v=rDCTXaNmypI Full acquisition: https://youtu.be/LRmJn2BAYwc
```

Comment:
the original is irrelevant now and doesn't belong in the db

Reply:
Removed the old video. Do you think the nodes in this room have to be re-arranged to account for the new trick? I am asking because it is technically possible to forgo the drone doing the IS. But the connector still connects the Security Drone event to the item. I added the two different ways to use IS in this room in a new commit with the discussed knowledge level for review. During my testing the difference between destroying the drone and not destroying the drone was 400 damage taken. The maximum amount of damage taken was 656 and 256 respectively, so I added 5% on top as the minimum requirement.

Reply:
Yeah, feel free to re-arrange, but make sure this path through the drone still remains. Your path for skipping the drone should be `Knowledge (Advanced)`.

Reply:
I didn't skip the drone event for now. Not activating the drone at all isn't very easy and isn't shown in the video. It will stay like this for now.